lib/registrant.rb

```diff
@@ -0,0 +1,23 @@
+class Registrant
+
+    attr_reader :name,
+                :age,
+                :permit,
```

Comment:
Careful here, we actually do not want to include permit in the attr_readers. We have a method called `permit?` which acts as an attr_reader, but follows our convention of having a `?` at the end since the method returns a boolean, so we do not want this extra method here.